docs/user-guide/configure-map.md

```diff
@@ -0,0 +1,52 @@
+
+<img src="../img/configure-map/configure-map.jpg" class="ms-docimage"/>
+
+## Layers 
+
+The Map Inline Editor opens with the Layers section available, where it is possible to edit the layers settings (by selecting a layer in the TOC) and visibility of layers present in the map:
+
+<img src="../img/configure-map/layers-conf.jpg" class="ms-docimage"/>
+
+* Control the layer transparency by scrolling left and right the transparency bar <img src="../img/button/transparency-bar.jpg" class="ms-docbutton" style="max-height:20px"/>.
+
+* Toggle the layer visibility by switching off <img src="../img/button/eyeoff.jpg" class="ms-docbutton"/> and on <img src="../img/button/eyeon.jpg" class="ms-docbutton"/> the "eye" icon.
+
+## Setting 
+
+The *Setting section* allows the user to:
+
+<img src="../img/configure-map/setting-conf.jpg" class="ms-docimage"/>
+
+* Enable/disable the **Zoom in/out** on the map
+
+* Change the position of the **Zoom in/out** by choosing one of the options available in the dropdown menu
+
+<img src="../img/configure-map/position-zoom.jpg" class="ms-docimage"/>
+
+* Enable/disable the **Pan interaction** on the map
+
+## Advanced map editor
+
+Inside the Configure Map Toolbar the **Advanced map editor** button <img src="../img/button/edit_button.jpg" class="ms-docbutton"/>is also available allow advanced customization to the map: clicking on that button, a MapStore viewer opens for this purpose.
+
+<img src="../img/configure-map/ad-edit-map.jpg" class="ms-docimage"/>
+
+The available tools to modify the map are the following:
+
+* Adding the **Layers** by using the <img src="../img/button/catalog-option.jpg" class="ms-docbutton" style="max-height:20px;"/> button in the *Option menu* <img src="../img/button/burger.jpg" class="ms-docbutton"/> as it is explained in the [Catalog Services](catalog.md).
+
+* Adding  **Annotations** by clicking on the <img src="../img/button/annotation-option.jpg" class="ms-docbutton" style="max-height:20px;"/> button in the *Option menu* <img src="../img/button/burger.jpg" class="ms-docbutton"/> as it is explained in the [Adding Annotations](annotations.md).
+
+* **Import** a map by clicking on the <img src="../img/button/import-button.jpg" class="ms-docbutton" style="max-height:20px;"/> button in the *Option menu* <img src="../img/button/burger.jpg" class="ms-docbutton"/> as it is explained in the [Import files](import.md).
+
+* **Change Background** as it is explained in the [Background Selector](background.md)
+
+* **Edit Layers** by clicking on the Layers button <img src="../img/button/show-layers.jpg" class="ms-docbutton"/> as it is explained in the [Table of Contents](toc.md)
+
+Once the changes have been made it is possible click on **Apply** to see the final result in the story.
```

Comment:
Once the advanced map editing is complete, it is possible click on **Apply** to see the final result in the story.